app/src/main/java/com/example/campusguide/map/GoogleMapInitializer.kt

```diff
@@ -28,27 +31,35 @@ class GoogleMapInitializer constructor(
 
     override fun onMapReady(map: GoogleMap?) {
         if (map != null) {
+            googleMap = map
             wrapper.adapted = map
             map.uiSettings.isMyLocationButtonEnabled = false
 
             // Center the map on Hall building
-            if (mapId == "maps_activity_map") {
+            //if (mapId == "maps_activity_map") {
                 val hall = LatLng(45.497290, -73.578824)
                 map.animateCamera(
                     CameraUpdateFactory.newLatLngZoom(
                         hall,
                         Constants.ZOOM_STREET_LVL
                     )
                 )
-            }
+            //}
             BuildingHighlights(map, activity).addBuildingHighlights()
 
             if(onPolygonClickListener != null) {
                 map.setOnPolygonClickListener(onPolygonClickListener)
             }
             map.setContentDescription("$mapId ready")
+
+            onMapReadyListener?.invoke()
```

Comment:
I think .invoke() replaceable by ()

Reply:
This is the only way to invoke if the variable is nullable, replacing `?.invoke()` by `()` or by `?()` does not work.